src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -36,7 +36,7 @@ export class OpenSheetMusicDisplay {
             throw new Error("Please pass a valid div container to OpenSheetMusicDisplay");
         }
 
-        if (options.backend.toLowerCase() === "svg" || options.backend === undefined) {
```

Comment:
Haha how could this even have worked? :D

Reply:
just a whoopsie daisy ;)